webnn/resources/utils.js

```diff
@@ -67,21 +67,21 @@ const loadTests = (operationName) => {
 };
 
 /**
- * Get exptected data and data type from given resources with output name.
+ * Get exptected sub output resources from given resources with output name.
```

Comment:
```suggestion
 * Get expected suboutput resources from given resources with output name.
```

Reply:
>  "Suboutput" to me implies a subspan of one of the outputs (a range of sliced elements), whereas this appears to just be grabbing one of the outputs.

That's correct. Sorry for missing your comment, will submit commit to update it.

Reply:
Yes，I'm working to fix it.

Reply:
How about close it for not trouble others.
And new one pr.

Reply:
Okie dokie. Well, I'll look at it Monday (just briefly checked my GitHub notifications before bed).